core/domain/story_domain.py

```diff
@@ -104,85 +108,132 @@ class StoryChange(change_domain.BaseChange):
 
     # The allowed list of story properties which can be used in
     # update_story_property command.
-    STORY_PROPERTIES = (
-        STORY_PROPERTY_TITLE, STORY_PROPERTY_THUMBNAIL_BG_COLOR,
+    STORY_PROPERTIES: List[str] = [
+        STORY_PROPERTY_TITLE,
+        STORY_PROPERTY_THUMBNAIL_BG_COLOR,
         STORY_PROPERTY_THUMBNAIL_FILENAME,
-        STORY_PROPERTY_DESCRIPTION, STORY_PROPERTY_NOTES,
-        STORY_PROPERTY_LANGUAGE_CODE, STORY_PROPERTY_URL_FRAGMENT,
-        STORY_PROPERTY_META_TAG_CONTENT)
+        STORY_PROPERTY_DESCRIPTION,
+        STORY_PROPERTY_NOTES,
+        STORY_PROPERTY_LANGUAGE_CODE,
+        STORY_PROPERTY_URL_FRAGMENT,
+        STORY_PROPERTY_META_TAG_CONTENT
+    ]
 
     # The allowed list of story node properties which can be used in
     # update_story_node_property command.
-    STORY_NODE_PROPERTIES = (
+    STORY_NODE_PROPERTIES: List[str] = [
         STORY_NODE_PROPERTY_DESTINATION_NODE_IDS,
         STORY_NODE_PROPERTY_ACQUIRED_SKILL_IDS,
-        STORY_NODE_PROPERTY_PREREQUISITE_SKILL_IDS, STORY_NODE_PROPERTY_OUTLINE,
-        STORY_NODE_PROPERTY_EXPLORATION_ID, STORY_NODE_PROPERTY_TITLE,
+        STORY_NODE_PROPERTY_PREREQUISITE_SKILL_IDS,
+        STORY_NODE_PROPERTY_OUTLINE,
+        STORY_NODE_PROPERTY_EXPLORATION_ID,
+        STORY_NODE_PROPERTY_TITLE,
         STORY_NODE_PROPERTY_DESCRIPTION,
         STORY_NODE_PROPERTY_THUMBNAIL_BG_COLOR,
-        STORY_NODE_PROPERTY_THUMBNAIL_FILENAME)
+        STORY_NODE_PROPERTY_THUMBNAIL_FILENAME
+    ]
 
     # The allowed list of story content properties which can be used in
     # update_story_contents_property command.
-    STORY_CONTENTS_PROPERTIES = (INITIAL_NODE_ID, NODE, )
+    STORY_CONTENTS_PROPERTIES: List[str] = [INITIAL_NODE_ID, NODE]
```

Comment:
If we use a tuple here instead of a list, then we have to define a Tuple type like this:

```
STORY_CONTENTS_PROPERTIESType = Tuple[
    Literal['initial_node_id'],
    Literal['node']
]
```
and assign it to the `STORY_CONTENTS_PROPERTIES` and this is just for STORY_CONTENTS_PROPERTIES, if we use a tuple we have to define a different tuple type for `STORY_PROPERTIES` and `STORY_NODE_PROPERTIES`. 

I think this is already resolved under comment ( https://github.com/oppia/oppia/pull/14958/files/13584164e61146528fe625473d19c11ea57dbde7#r811968163 )